app/assets/stylesheets/letter.scss

```diff
@@ -0,0 +1,37 @@
+$oil: #333333;
+$white: #ffffff;
+
+.letter {
+  background: $white;
+  box-shadow: 0 0 10px rgba(0,0,0,0.3);
+  margin: 26px auto 0;
+  width: 90%;
+  min-height: 300px;
+  padding: 24px;
+  position: relative;
+
+  table {
+    border: none;
+  };
+}
+.letter:before, .letter:after {
+  content: "";
+  height: 98%;
+  position: absolute;
+  width: 100%;
+  z-index: -1;
+}
+.letter:before {
+  background: #fafafa;
```

Comment:
Color literals like `#fafafa` should only be used in variable declarations; they should be referred to via variable everywhere else.